jdk/src/solaris/bin/java_md_solinux.c

```diff
@@ -380,6 +386,13 @@ CreateExecutionEnvironment(int *pargc, char ***pargv,
                                            asked for? Current model is
                                            fine unless another model
                                            is asked for */
+#ifdef AIX
+    static const char *mallocOptinsName = "MALLOCOPTIONS";
+    static const char *mallocOptinsValue = "multiheap,buckets";
+    if (setenv(mallocOptinsName, mallocOptinsValue, 0) != 0) {
+      printf("mallocOptins failed\n");
+    }
```

Comment:
The coding style in this repo uses 4 spaces per indentation: please follow suit.

More importantly, this will over-write a user's environment variable: I think this should only set `MALLOCOPTIONS` if it is not already set. I don't think `printf` is the proper response to a `setenv` failure.

Each variable is missing an 'o' in the word 'options'; the should also be declared `static const char []` - we don't need the pointers too.

Reply:
@keithc-ca I don't think `setenv(mallocOptinsName, mallocOptinsValue, 0)`, passing 0 for the overwrite parameter, does overwrite an existing variable.

@shanchao95 note we don't need static vars, as setenv() makes copies of the strings.

Reply:
Perhaps I was thinking of `putenv` which always overwrites existing values.

I'm fine with inlining the string constants, but if we keep the variables, they should be
```
static const char mallocOptionsName[] = "MALLOCOPTIONS";
static const char mallocOptionsValue[] = "multiheap,buckets";
```